blog-service/2023-03-28-otel-collector-apps.md

```diff
@@ -0,0 +1,20 @@
+---
+title: NEW - Get Started with OpenTelemetry Monitoring in Minutes
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - opentelemetry
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We’re happy to announce our new and improved OpenTelemetry collector data onboarding workflow that gets you up and running with infrastructure monitoring in minutes. With this update, you can start monitoring host and process data, web servers (like IIS, Nginx), databases (like MySQL, Redis, Cassandra), and other sources out of the box - no manual configuration required. [Learn more](/docs/get-started/quickstart).
```

Comment:
```suggestion
We’re happy to announce a release that saves you configuration time. Our new and improved OpenTelemetry collector data onboarding workflow that gets you up and running with infrastructure monitoring in minutes. With this update, you can start monitoring host and process data, web servers (like IIS, Nginx), databases (like MySQL, Redis, Cassandra), and other sources out of the box - no manual configuration required. [Learn more](/docs/get-started/quickstart).
```

Reply:
Your text is great but I feel like you're burying the lede. This helps the user get started much faster.